signale.js

```diff
@@ -257,6 +257,19 @@ class Signale {
     this._log(this._buildSignale(this._types[type], ...messageObj), this._types[type].stream);
   }
 
+  _padSpace(str, targetLength) {
+    if (String.prototype.padEnd) {
+      return str.padEnd(targetLength);
+    }
+    targetLength >>= 0; // Floor if number or convert non-number to 0;
+    if (str.length > targetLength) {
+      return String(str);
+    }
+    targetLength -= str.length;
+    const padString = ' '.repeat(targetLength);
```

Comment:
could be simplified to just `const pad = ' '.repeat(targetLength);` or even `return str + ' '.repeat(targetLength);`

Reply:
I would prefer `return str + ' '.repeat(targetLength);`